packages/wrangler/src/metrics/metrics-config.ts

```diff
@@ -174,6 +155,14 @@ export interface MetricsConfigFile {
 		enabled: boolean;
 		/** The date that this permission was set. */
 		date: Date;
+		/** Version number the banner was last shown - only show on version update */
+		bannerLastShown?: string;
+	};
+	c3permission?: {
+		/** True if c3 should send metrics to Cloudflare. */
+		enabled: boolean;
+		/** The date that this permission was set. */
+		date: Date;
```

Comment:
Why are we storing c3 metrics here?

Reply:
c3 and wrangler permissions were saved in the same file :/